test/test_json_response.rb

```diff
@@ -20,6 +20,12 @@
       app.call(RC::RESPONSE_BODY => '{}') do |response|
         response.should.eq(expected)
       end
+
+      expected = {RC::RESPONSE_BODY => nil,
+                  RC::REQUEST_HEADERS => {'Accept' => 'application/json'}}
+      app.call({}) do |response|
+        response.should.eq(expected)
+      end
```

Comment:
Thanks, I think the logic is good here, but let's put this in a separate test, maybe something like:

``` ruby
would 'not decode but just return nil if response body is nil' do
  # ...
end
```

Reply:
ok, done.